packages/vite/src/node/plugins/preAlias.ts

```diff
@@ -14,7 +14,11 @@ export function preAliasPlugin(): Plugin {
       server = _server
     },
     resolveId(id, importer, options) {
-      if (!options?.ssr && bareImportRE.test(id)) {
+      // When using absolute path to import dep instead of bare import, the dep properly has pre-bundled. #5494
+      if (
+        !options?.ssr &&
+        (bareImportRE.test(id) || server._optimizeDepsMetadata?.optimized[id])
```

Comment:
@sanyuan0704 you probably need this here:
```suggestion
        (bareImportRE.test(id) || server._optimizeDepsMetadata?.optimized[normalizePath(id)])
```

Reply:
Thanks, i will have a try.

Reply:
I tried it, if using `define` can not make the test project run, so I still use `resolve.alias`, but as you suggest, i replaced `{{ F }}` with `__F_ABSOLUTE_PACKAGE_PATH__`.

Reply:
It does not seem to be a problem with normalizePath. I have added normalizePath but the problem still exists in windows environment.